middlewares/ensure_authenticated_user.go

```diff
@@ -3,12 +3,13 @@ package middlewares
 import (
 	"net/http"
 
-	"github.com/gin-gonic/gin"
 	session "github.com/gutakk/go-google-scraper/helpers/session"
+
+	"github.com/gin-gonic/gin"
 )
 
 func EnsureAuthenticatedUser(c *gin.Context) {
-	userID := session.Get(c, "user_id")
+	userID := c.MustGet("currentUser")
```

Comment:
What is the benefit of using `MustGet` over `session.Get`?

Reply:
`session.Get` will get the cookie from browser directly, which I had it in the `current_user` middleware. `current_user` middleware is the first step of the route validation and also had a logic to check for user in the database with `user_id` and store the `currentUser` to gin context.

`MustGet` is the function to get value from gin context from specific key.